src/wavesurfer.js

```diff
@@ -36,6 +36,7 @@ var WaveSurfer = {
         mediaType     : 'audio',
         autoCenter    : true
     },
+    tempUrl           : null,
```

Comment:
This property will be defined on the prototype. Every time you assign a value, it will be shared across all wavesurfer instances.